crates/cheatcodes/src/inspector.rs

```diff
@@ -1826,10 +1845,7 @@ impl Cheatcodes {
     fn meter_gas(&mut self, interpreter: &mut Interpreter) {
         if let Some(paused_gas) = self.gas_metering.paused_frames.last() {
             // Keep gas constant if paused.
-            // Make sure we record the memory changes so that memory expansion is not paused.
-            let memory = interpreter.control.gas.memory;
             interpreter.control.gas = *paused_gas;
-            interpreter.control.gas.memory = memory;
```

Comment:
Not entirely clear whether we should keep or remove this

`interpreter.control.gas.memory`

now yields

```
use of deprecated method `revm::revm_interpreter::Gas::memory`: memory expansion cost is not tracked anymore; calculate it using `SharedMemory::current_expansion_cost` instead
`#[warn(deprecated)]` on by default
```